### DISCUSSION TITLE
Add a way to create/edit commands in chat

### DISCUSSION THREAD

**Discussion by hibsbsbs** [Chatterino/chatterino2#3699] [Ideas]
Add a way to create/edit commands directly in chat without having to open the settings

e.g. /command add "trigger" "cmd message {1+}" 
would create the command
![image](https://user-images.githubusercontent.com/104651861/166058701-8798355e-f556-49a5-8996-d65377f1a948.png)

similarly, a way to edit a command:

/command edit "trigger" "edited cmd"
would edit the already existing command "trigger" to match the input
![image](https://user-images.githubusercontent.com/104651861/166059109-df346381-f57a-4ce8-9e19-18be1c1d11f0.png)

also, pretty self explanatory, add a way to delete commands:
/command delete "trigger"

**Comment by Mm2PL** [comment]
In what practical way would this be useful? While it looks cool, I can't really think of one myself.

**Reply by hibsbsbs** [reply]
Honestly sometimes I don't want to open the settings JUST to add a command (it doesn't help that my Chatterino freezes everytime I open settings for some reason), and it would just be faster this way. Other than that, it probably isn't that useful.
